src/transport/BLE.h

```diff
@@ -84,12 +84,24 @@ class BleConnectionParameters
         return *this;
     }
 
+    bool HasMacAddress() const { return mHasMacAddress; };
+    const uint8_t (&GetMacAddress())[6] { return mMacAddress; };
```

Comment:
Please add a using declaration of typedef here:

using BLEMacAddress = uint8_t [6];

const BLEMacAddress& GetMacAddress() const;